lib/src/blocs/app/notification_cubit.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:convert';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import 'package:http/http.dart' as http;
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import 'notification_state.dart';
+
+class NotificationCubit extends Cubit<NotificationStatusState> {
```

Comment:
why do we have both a Bloc and a Cubit for the notifications, esp if all the Cubit is doing is calling the Bloc?

Reply:
I don't think there is a notification bloc. The cubit sends the notification toggle state and sends the fcm notification